### PR TITLE
add apim sbox

### PR DESCRIPTION
**Pull request by thomast1906**
Add apim sbox

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by thomast1906** [comment]
Don't review atm, testing

**Review by timja — APPROVED** [review]

File: components/apim/init.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">= 0.15.4"
```
Comment (timja) [inline components/apim/init.tf:2]: use greater than 1.0.0?

Comment (thomast1906) [reply components/apim/init.tf:2]: It did error with 1.0.0 due to current setting to 0.15.4, I suppose upgrading to 1.0.0 should be fine

Comment (thomast1906) [reply components/apim/init.tf:2]: Resolved now @timja

**Review by timja — APPROVED** [review]

**Pull request merged**